### PR TITLE
Add GitHub Action workflow

### LINKED ISSUE #25: Add CI
- Add CI
  - Use GitHub Actions

### PR DESCRIPTION
**Pull request by enokawa**
Closes: #25 

## Description

- add GitHub Actions workflow

## Details of this change

- add frontend workflow

## Information for Reviewer

- [Workflow syntax for GitHub Actions](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions)
- [Using Node.js with GitHub Actions](https://docs.github.com/en/actions/language-and-framework-guides/using-nodejs-with-github-actions)

### CODE REVIEW

**Comment by enokawa** [comment]
@nari-z Prealse review. Maybe it will work.

File: .github/workflows/frontend.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          ${{ runner.OS }}-
+
+    - name: Install dependencies
+      run: yarn --frozen-lockfile
+
+    - name: Build
+      run: yarn build
+
+    # - name: Test
+    #   run: yarn test
```
Comment (nari-z) [inline .github/workflows/frontend.yml]: How about adding a `yarn lint`?

File: .github/workflows/frontend.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      uses: actions/cache@v2
+      with:
+        path: ~/.npm
+        key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.OS }}-node-
+          ${{ runner.OS }}-
+
+    - name: Install dependencies
+      run: yarn --frozen-lockfile
```
Comment (nari-z) [inline .github/workflows/frontend.yml:43]: 👍

**Review by nari-z — CHANGES_REQUESTED** [review]

File: .github/workflows/frontend.yml (hunk 1 above)
Comment (enokawa) [reply .github/workflows/frontend.yml]: Sounds great!!

Comment (enokawa) [reply .github/workflows/frontend.yml]: Added: 01ceb7781b30bddad1f1090b488b4d62685ca4cb .

**Review by nari-z — APPROVED** [review]
LGTM

**Pull request merged**